docs/rfcs/0001-congratulations-letter-to-dashboard.md

```diff
@@ -0,0 +1,39 @@
+## Add Congratulations Letter to Dashboard
+
+### Abstract
+
+As a learner who has earned a program certificate, I sometimes have trouble explaining the meaning and value of the certificate. It's useful to have a letter from the faculty that describes my accomplishment that I can share with employers, in addition to the certificate.
+
+### Architecture Changes
+
+We have already codebase for program certificate creation, logic of creating program congratulation is more or less same as program certificate creation.
+
+#### Database:
+We can add new model `MicromastersProgramCompletion` in `grades/models.py` with the following fields.
```

Comment:
Suggested model name: `MicromastersProgramCommendation`

Reply:
"Commendation" works for me. "Letter" is too generic and "Accomplishment" is too vague.

Reply:
already updated.